test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,48 @@
+LD_FLAGS=$1
+IMAGE_VERSION=$2
+EKS_CLUSTER_NAME=$3
+
+AWS_REGION="us-west-2"
+AWS_ACCOUNT_ID=$(aws --region $AWS_REGION sts get-caller-identity --query Account --output text)
+
+ECR_REGISTRY_NAME="csisecretdriver"
+ECR_REGISTRY_URL="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com"
+ECR_REGISTRY_URL_WITH_NAME="$ECR_REGISTRY_URL/$ECR_REGISTRY_NAME"
+
+ECR_IMAGE_NAME="csisecretdriver"
+NAMESPACE="kube-system"
+
+AWS_SERVICE_ACCOUNT_NAME="basic-test-mount-sa"
+
+docker build  --no-cache --build-arg LDFLAGS="$LD_FLAGS" -t $ECR_IMAGE_NAME -f ./docker/Dockerfile .
```

Comment:
Having this command repeated here will make it difficult to keep consistent across `Makefile` and other occurrences. Could we just using the makefile target to build and push image? We do that today for the azure windows jobs.

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/8b9f6a2784f33fad78cd97cf3234921a6ac9959b/Makefile#L282

Similarly, just `make container-linux` after authenticating with the registry should be sufficient.

Reply:
The `make container-linux` will need to be run after authenticating with the aws registry